.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 1

```

I thought this should always be `2`.

```suggestion
version: 2
```

---

got it , you are right it supports only the version 2 and fails with 1